.github/workflows/linters.yml

```diff
@@ -0,0 +1,1789 @@
+
```

Comment:
You did not set up your linters correctly. Please follow [this documentation](https://github.com/microverseinc/linters-config/blob/master/html-css/README.md) to set them up correctly.